### DISCUSSION TITLE
Disable regular `cd` behavior

### DISCUSSION THREAD

**Discussion by rcorre** [ajeetdsouza/zoxide#625] [Q&A] [answered]
If I type `z foo` and there is a local directory named "foo", it will `cd` to that local directory, even if there is a directory named "foo" in the database with a higher weight. Is there a way to disable this behavior, and have `z` always select an entry from the database?

**Comment by ajeetdsouza** [comment, marked as answer]
Refer to https://github.com/ajeetdsouza/zoxide/issues/852.
